Documentation/HowToGuides/CreatingProxyActionsAndChainingActions/README.md

```diff
@@ -0,0 +1,82 @@
+
+![Deactivating The Boolean](assets/images/DeactivatingTheBoolean.png)
+
+### Step 4
+
+Now we have a generic Action that will disable the `Floor` GameObject when the Action receives `true` and will re-enable the `Floor` GameObject when the Action receives `false`.
+
+We could hook up the `Unity Button Action` `Value Changed` event to simply call the `Receive` method on our `FloorToggler` GameObject `BooleanAction` component and this will pass the state of the `Unity Button Action` over to the `Boolean Action` whenever the value changes. However, doing this means for every input action, we’d need to wire up their `Value Changed` event to point to the `Receive` method on our `BooleanAction` component. This can again get very cumbersome and hard to maintain, so instead we’re going to use the `Sources` parameter that is available on a `Zinnia.Action.`
+
+The `Sources` parameter allows us to specify other Actions for our `BooleanAction` to monitor changes on and re-emit any events that happen on those source actions.
+
+Expand the `Sources parameter` and enter the desired number of other Actions to monitor in the `Size` parameter field. In this example, we want to watch two other Actions (the Menu Button `Unity Button Actio` and the Space Key `Unity Button Action`) so enter `2` into the Sizes parameter field.
+
+![Update Sources Parameter On Boolean Action](UpdateSourcesParameterOnBooleanAction.png)
+
+### Step 5 
+
+Drag and drop the ButtonA_Press[0] MenuButton -> Press[0] GameObject into the `Sources -> Size -> Element 0` slot on the `FloorToggle`r GameObject `BooleanAction` component. This will set the `Unity Button Action` component on the Press[0] GameObject as a Source on the `BooleanAction` component on the `FloorToggler` GameObject.
+
+Drag and drop the `GameObject` GameObject that contains the `Unity Button Action` component that is working with the Space Key into the `Sources -> Size -> Element` 1 slot on the `FloorToggler` GameObject `BooleanAction` component as well.
+
+!{Drag and Drop Source Actions] (DragAndDropSourceActions.png)
+
+### Step 6 
+
+The`Unity Button Action` events still have actions to perform when either the Menu Button is pressed or the Space Key is pressed, we no longer need those actions on those `Unity Button Action` components now as they are being chained to our new FloorToggler GameObject `BooleanAction` component. So let’s remove the event listeners from the other Unity Button Action components.
+
+Select the Input.UnityInputManager.XboxController -> ButtonA_Press[0] GameObject and click the `-` symbol in the bottom right corner of the `Activated` event parameter and the `Deactivated` event parameter on the `Unity Button Action` component. This will remove those event listeners and make this `Unity Button Action` component an Action that on its own does nothing other than listen for button presses on the Menu Button.
```

Comment:
add backticks to: Input.UnityInputManager.XboxController -> ButtonA_Press[0]

so it's `Input.UnityInputManager.XboxController -> ButtonA_Press[0]`

Again `Unity Button Action` is now `Unity Input Manager Button Action` so update any references.

Also, any reference to `Menu Button` no longer makes sense as this is from the old example.

The new example should be referencing the Space key and the Xbox Controller A button